lib/xcake/project/sugar.rb

```diff
@@ -0,0 +1,102 @@
+require 'xcodeproj'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.